### ISSUE TITLE
Suboptimal edit?

### ISSUE THREAD

**Issue by eneroth** [juji-io/editscript#16]
In this diff:

```clojure 
(def a [:zero {:x :y}])
(def b [:zero {:a "a" :b "b" :c "c" :d "d" :e "e"}])

(diff a b))
 
=> [[[1] :-] [[1] :+ {:a "a", :b "b", :c "c", :d "d", :e "e"}]]
```

… it seems like removing and adding would be suboptimal. Wouldn't replacing be a smaller solution?
```clojure 
[[[1] :r {:a "a", :b "b", :c "c", :d "d", :e "e"}]]
````

**Comment by huahaiy** [comment]
Thanks for reporting. I will look into this later.

**Comment by huahaiy** [comment]
Fixed in 0.4.4
